client/src/App.js

```diff
@@ -0,0 +1,28 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import ContextState from './contextState/contextState';
+import Home from './pages/home/Home';
+import Register from './pages/register/Register'
+import Post from './pages/posts/Post';
+import Comment from './pages/comments/Comments';
+import NotFound from './pages/notFound/NotFound';
+
+function App() {
+
+  return (
+    <div className='App-header'>
+      <ContextState>
+        <Router>
+          <Routes>
+            <Route exact path='/' element={<Home />} />
```

Comment:
Separate the protected routes.

Reply:
I made them protected by using the condition that if the user is not logged in, it will redirect to home page.
Fixed